packages/@aws-cdk/aws-redshift-alpha/README.md

```diff
@@ -543,3 +543,19 @@ const cluster = new Cluster(this, 'Redshift', {
 });
 cluster.addIamRole(role);
 ```
+
+## Resizing
+
+As your data warehousing needs change, it's possible to resize your Redshift cluster, but it's important to understand certain caveats.
+Most importantly, if the cluster was deployed via CDK, the cluster should be resized via CDK so it's registered in the AWS CloudFormation template.
+This ensures the `NumberOfNodes` parameter gets updated so the correct type of resize is done. There are two types of resize operations:
```

Comment:
CFN has this as `NumberOfNodes`, CDK has it as `numberOfNodes`. I felt it best to be `NumberOfNodes` to represent the CFN template.

Reply:
I think this whole sentence can be removed. Just noting that the resize has to be done via CloudFormation is a good reminder, no need to go into detail. You are saying "certain caveats". Are there more than one?

As for your casing example: as a rule, I would say that CDK documentation should stay in CDK-land, and not presume CloudFormation knowledge. I know this is not entirely achievable always, but that should be our goal. The point is moot though, again, I would remove the sentence.